phases/snapshot/docs/wasi_unstable_preview1.md

```diff
@@ -2286,10 +2286,6 @@ Members:
 
     - <a href="#subscription.u.clock" name="subscription.u.clock"></a>**`u.clock`**
 
-        - <a href="#subscription.u.clock.identifier" name="subscription.u.clock.identifier"></a><code>[\_\_wasi\_userdata\_t](#userdata) <strong>identifier</strong></code>
-
-            The user-defined unique identifier of the clock.
-
         - <a href="#subscription.u.clock.clock_id" name="subscription.u.clock.clock_id"></a><code>[\_\_wasi\_clockid\_t](#clockid) <strong>clock\_id</strong></code>
```

Comment:
```suggestion
        - <a href="#subscription.u.clock.clock_id" name="subscription.u.clock.clock_id"></a><code>[\_\_wasi\_clockid\_t](#clockid) <strong>id</strong></code>
```

Reply:
This "hotfix" rename matches the current state of witx, see [typenames.witx#L558](https://github.com/WebAssembly/WASI/blob/master/phases/snapshot/witx/typenames.witx#L558), where `clock_id` field was renamed to `id`.

Reply:
The links should be fixed as well, but I don't wanna clutter this PR so I reckon we can do it in a subsequent PR, or when we come up with a more automatic strategy for generating nicely formatted docs directly from the witx files.

Reply:
I think we should work on automatic generation rather than try to maintain this html/md by hand.

Reply:
Agreed! I’ve got a draft solution already in the works, and hopefully should have it ready today/tomorrow! :-)